README.md

```diff
@@ -33,7 +33,7 @@ JDK 1.8.
 
 ## Supported OnPrem vCenter Releases:
```

Comment:
I think we need to add a section which VMC M / NSX releases are supported as well, e.g., VMC M 18 , etc.

I don't understand what this section means. Is it up-to-date?

"VMware Cloud on AWS Support:

The VMware Cloud on AWS API and samples are currently available as a preview and are subject to change in the future."